### PR TITLE
Fix 144

### PR DESCRIPTION
**Pull request by nonara**
## Related Issues

- #144 
- https://github.com/crosstype/node-html-markdown/issues/25

## Still Failing

- Deeply nested A

   ```html
   <a href="#"><b>link <a href="#">nested link</a> end</b></a>
   ```

- Uppercase A tag name

   ```html
   <A href="#"><b>link <a href="#">nested link</a> end</b></A>
   ```

## Performance Notes

When I said it wouldn't have a perf impact, I should have more correctly stated that the impact would be negligible.

I did a bit of perf testing and found that the added operation: `if (myVar === 'a' || myVar === 'A')` runs at 436,132,968 ops per second. The extra bit of logic inside the conditional will be even less impactful, as it will rarely execute, and is also extremely inexpensive.

In short, I don't expect a noticeable difference for the changes.

### CODE REVIEW

**Comment by nonara** [comment]
@taoqf Looks like you've got a bit going on. Would you be willing to make me a maintainer so I can help keep things moving and address some of the issues?

I understand the priority of speed, and I maintain several large / critical repos.

**Comment by taoqf** [comment]
@nonara Happy to hear this.

**Comment by taoqf** [comment]
Busy is not the only thing stop me, but also the gfw. I am not access to github for month.

**Comment by nonara** [comment]
Thanks! Will be glad to help out. Sorry to hear about that. I didn't realize the GFW was blocking GitHub!
